packages/sui-bundler/README.md

```diff
@@ -54,6 +56,29 @@ $ sui-bundler dev
 
 While developing your app, you will have HMR (Hot Module Reloading). Default port for your website is 3000, but it will assign automatically a free port for you if this one is busy.
 
+#### How to link packages
+
+`sui-bundler` give us the oportunity to link `npm` packages with `--link-package`. This argument accepts relative and absolute paths.
+The `npm link` is cancelled as soon as we stop/cancel the terminal process.
```

Comment:
it sounds like `npm link` is executed but it is not, it just adds the code to the bundle you are working on. Maybe you could write that as soon as we stop/cancel the process, the packages are not linked anymore or something like that

Reply:
Yes, that true. Looks like internally is executing `npm link`. I'll update it